spec/requests/status_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
```

Comment:
Can't think of a way to avoid these - long blocks are pretty common in specs so it's going to be recurring annoyance. Just wondering if there's a way to do this with a global rule for `*_spec.rb` files. Could we have a `.rubocop.yml` in the spec directory?

Reply:
Yep - good shout

Reply:
Add this to rubocop.yml

```
Metrics/BlockLength:
  Enabled: false
```

Reply:
This works in the root `.rubocop.yml` I think ...

```
Metrics/BlockLength:
  Exclude:
    - 'spec/**/*'
```